astropy/coordinates/sky_coordinate.py

```diff
@@ -951,7 +951,7 @@ def __dir__(self):
         """
         Override the builtin `dir` behavior to include:
         - Transforms available by aliases
-        - Attribute / methods of the underlying self.frame object
+        - Attribute / methods of the underlying self.frame object.
```

Comment:
With a full stop here we probably want a `;` on the line above.

Reply:
I added a summary line.